ports/servoshell/desktop/events_loop.rs

```diff
@@ -62,7 +63,7 @@ impl EventsLoop {
         Ok(EventsLoop(if headless {
             EventLoop::Headless(Arc::new((Mutex::new(false), Condvar::new())))
         } else {
-            let mut event_loop_builder = winit::event_loop::EventLoopBuilder::with_user_event();
+            let mut event_loop_builder = winit::event_loop::EventLoop::with_user_event();
```

Comment:
Can you also do a use `winit::event_loop::EventLoop` above and replace all the of the qualified usage in this file? It's odd to mix them. I know that the existing code does this, but we should start cleaning it up.

```suggestion
            let mut event_loop_builder = EventLoop::with_user_event();
```

Reply:
I guess it was done because there's already the `EventLoop` enum in scope